File: modulo1/segmento2/apunte1/ejercicio2.web/ejercicio2.js
function datos(){

		const nombre = prompt ('Ingrese su nombre: ');
		alert(`Hola: ${nombre}`);

		const apellido = prompt ('Ingrese su apellido: ');
		alert(`Tu apellido es: ${apellido}`);

		const dni = prompt ('Ingrese su numero de documento: ');
		alert(`Su numero de documento es: ${dni}`);

		const direccion = prompt ('Ingrese su domicilio: ');
		alert(`Su direccion es:  ${direccion}`);

		const telefono = prompt ('Ingrese su numero telefonico');
		alert(`Su numero de telefono es: ${telefono}`);

		alert('Su nombre y apellido es: ' + nombre + ' ' + apellido + ' Su numero de documento es: ' + dni + ' Su direccion es: ' + direccion + ' Y su numero telefonico es: ' + telefono); 

}

File: modulo1/segmento2/apunte1/ejercicio2/Ejercicio2.java
import java.util.Scanner;
public class Ejercicio2
{
	public static void main(String args[])
	{
			Scanner keyboard = new Scanner(System.in);

			System.out.print("Ingrese su nombre: ");
			String name = keyboard.next();

			System.out.print("Ingrese su apellido: ");
			String LastName = keyboard.next();

			System.out.print("Ingrese su DNI: ");
			int dni = keyboard.nextInt();

			System.out.print("Ingrese su direccion: ");
			String address = keyboard.next();

			System.out.print("Ingrese su numero telefonico: ");
			int tel = keyboard.nextInt();

			System.out.println("Su nombre y apellido es: " + name + " " + LastName + " Su documento es: " + dni + " Su direccion es: " + address + " Y su numero telefonico es: " + tel);


	
	}

}
